analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -143,7 +148,10 @@ private ProgramState ProcessBranchState(ControlFlowBranch branch, ProgramState s
                     state = state.RemoveCapture(capture);
                 }
             }
-            return state.ResetOperations();
+
```

Comment:
This is quite a lot of symbols to separate two commands in a language 😄 
`}`, `newline`, `newline`

```suggestion
```